core/storage.js

```diff
@@ -31,12 +31,13 @@ define('core/storage',
     }
 
     function mapKey(key) {
-        return settings.storage_version + '::' + key;
```

Comment:
What was wrong with this again? I know we're not using it but don't know if hardcoding it changes much. Although ```settings.storage_version || '0'``` might've been better